genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/JobKickoffTask.java

```diff
@@ -162,20 +172,30 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
             } catch (final IOException ie) {
                 throw new GenieServerException("Unable to start command " + String.valueOf(command), ie);
             }
-            log.info("Finished Job Kickoff Task for job {}", jobExecEnv.getJobRequest().getId());
+            log.info("Finished Job Kickoff Task for job {}", jobRequest.getId());
```

Comment:
This will log out as [optional: etc]. Don't we already have the id somewhere referenced so we can just use it directly?